File: Java/Program/Shirt.java

class Shirt extends Clothing{
    private String color;
    private String sleeve_type;

    public Shirt(){

    }
    
    public Shirt(String color, String sleeve_type) {
        this.color = color;
        this.sleeve_type = sleeve_type;
    }

    public Shirt(String idproduct, String name, String brand, int price, String size, String material, String gender, String color, String sleeve_type) {
        super(idproduct, name, brand, price, size, material, gender);
        this.color = color;
        this.sleeve_type = sleeve_type;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public String getSleeve_type() {
        return sleeve_type;
    }

    public void setSleeve_type(String sleeve_type) {
        this.sleeve_type = sleeve_type;
    }

}

File: Python/Program/Shirt.py
from Clothing import Clothing


class Shirt(Clothing):
    __color = ""
    __sleeve_type = ""
    
    def __init__(self, color = "", sleeve_type = ""):
        self.__color = color
        self.__sleeve_type = sleeve_type
        
    def __init__(self, id, nama, brand, price, size, material, gender, color, sleeve_type):
        super().__init__(id, nama, brand, price, size, material, gender)  
        self.__color = color
        self.__sleeve_type = sleeve_type
        
    # Getter
    def get_color(self):
        return self.__color

    def get_sleeve_type(self):
        return self.__sleeve_type

    # Setter
    def set_color(self, color):
        self.__color = color

    def set_sleeve_type(self, sleeve_type):
        self.__sleeve_type = sleeve_type
